src/components/AgreementSections/ControllingLicense.js

```diff
@@ -52,7 +52,7 @@ export default class ControllingLicense extends React.Component {
 
   renderLicense = linkedLicense => {
     const currentAmendments = getLicenseAmendments(linkedLicense, statuses.CURRENT);
-    const unsetAmendments = getLicenseAmendments(linkedLicense, null);
+    const unsetAmendments = getLicenseAmendments(linkedLicense, statuses.UNSET);
```

Comment:
What this line is doing is so much more obvious now! 😍

Reply:
Cheers :)